scripts/generate_sudo_list.py

```diff
@@ -0,0 +1,74 @@
+"""Helper script to generate sudo list doc."""
+import argparse
+import filecmp
+import sys
+import tempfile
+from pathlib import Path
+
+import yaml
+
+
+def collect_sudo_commands(path=None):
+    """Collect all ansible cmds that require sudo."""
+    if not path:
+        path = Path("quipucords/scanner/network/runner")
+    list_cmds = []
+    for playbook in path.glob("roles/*/tasks/main.yml"):
+        for task in yaml.safe_load(playbook.open()):
+            if task.get("become", ""):
```

Comment:
just FYI, technically become could be "falsy", but since we don't do that I think it's OK to leave it as is.